build/build.proj

```diff
@@ -103,5 +104,47 @@
     <Error Text="There were test failures, for full log see %(HtmlTestFile.FullPath)" Condition="$(ExitCode) != 0" />
 
   </Target>
- 
+
+  <Target Name="TestIntegration" Condition="'$(RunIntegrationTests)' == 'true'">
+    <PropertyGroup>
+      <DownloadRoslynVsixesExe>$(NUGET_PACKAGES)\roslyntools.downloadroslynvsixes\0.3.2-beta\tools\DownloadRoslynVsixes.exe</DownloadRoslynVsixesExe>
```

Comment:
Can we put the package version numbers in a property so that there's one place to update? Same for xunit as well.